src/WebUI/src/components/moderator/CreateRestrictionForm.vue

```diff
@@ -52,63 +52,61 @@ const addRestriction = async () => {
 </script>
 
 <template>
```

Comment:
Here I just removed the wrapper `<div class="mb-12 space-y-8">`
as this component is now in the modal window

there are no more edits in this file